tes-serverd/src/http_server.c

```diff
@@ -72,7 +74,65 @@ static void request_handler(struct evhttp_request *req, void *arg) {
         }
         return;
     }
-    // TODO: Check the Authorization's value
+
+    GGL_LOGI("Auth header received: %s", auth_header);
```

Comment:
Is writing the auth header to logs okay? Seems like then anyone with access to read the logs can then spoof a request

Reply:
I can remove that . Good catch

Reply:
updated